packages/utils/telemetry-utils/src/config.ts

```diff
@@ -31,6 +38,8 @@ export interface IConfigProvider extends IConfigProviderBase {
  * Creates a base configuration provider based on `sessionStorage`
  *
  * @returns A lazy initialized base configuration provider with `sessionStorage` as the underlying config store
+ *
+ * @public
```

Comment:
I'd leave this internal. If we do export it I would see it as a "favor", which is probably not worth it in case someone takes a hard dependency on it and we want to make changes.  It's not hard to DIY this.

Reply:
+1. Confirmed no uses in bohemia nor loopsdk.